src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -135,6 +136,19 @@ public GroupDialog(DialogService dialogService, BasePanel basePanel, JabRefPrefe
             this.setTitle(Localization.lang("Edit group"));
         }
 
+        nameField.textProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
Can you please move these lines further down to https://github.com/JabRef/jabref/pull/4801/files#diff-2afbba630cb28bced7c30efa7c0a24dfR362, where similar listeners are registered for the other controls. Please also remove the `onAction` listener for these three controls as they are now obsolete.

Reply:
I'm really sorry that I didn't notice that there's someone else working on this. I have made the necessary changes you've requested. 
Regards,
Harinda